source/MaterialXGenShader/Util.cpp

```diff
@@ -588,6 +588,47 @@ bool hasElementAttributes(OutputPtr output, const StringVec& attributes)
     return false;
 }
 
+bool inputChangeRequiresShaderGen(InputPtr input, const string& target)
```

Comment:
Handle conditional node for inputs either directly on or connected to an interface input.